Libraries/zephyr/MAX/Include/wrap_max32_sys.h

```diff
@@ -69,13 +74,15 @@ static inline void Wrap_MXC_SYS_SetClockDiv(int div)
 #define ADI_MAX32_CLK_ISO MXC_SYS_CLOCK_ISO
 #endif
 
+#if !defined(CONFIG_SOC_MAX78002)
 #define z_sysclk_prescaler(v) MXC_SYS_CLOCK_DIV_##v
 #define sysclk_prescaler(v) z_sysclk_prescaler(v)
 
 static inline void Wrap_MXC_SYS_SetClockDiv(int div)
 {
     MXC_SYS_SetClockDiv((mxc_sys_system_clock_div_t)div);
 }
+#endif
```

Comment:
As per of UG I have, MAX78002 support system clock prescaler,
It might not be added in peripheral driver in that case you should add it in driver as https://github.com/analogdevicesinc/msdk/pull/1078
Then remove this if guard.
![image](https://github.com/user-attachments/assets/b3cdd0bf-fcee-4071-9cef-1b1d730a74b5)

![image](https://github.com/user-attachments/assets/d05c6c26-8452-44a8-b6e7-4b8b635cd1db)

Reply:
Opened https://github.com/analogdevicesinc/msdk/pull/1170 PR for this.